test-cases/performance/perf-regression.100threads.100M-keys-z3-enterprise.yaml

```diff
@@ -0,0 +1,61 @@
+test_duration: 1200
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=QUORUM duration=15m -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=IncrementalCompactionStrategy)' -rate threads=400 -mode cql3 native -col 'n=FIXED(5) size=FIXED(200)' -pop seq=1..6000M"
+prepare_write_cmd: ["cassandra-stress write no-warmup cl=QUORUM n=1000M -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=IncrementalCompactionStrategy)' -mode cql3 native -rate threads=300 -col 'n=FIXED(5) size=FIXED(200)' -pop seq=1..1000M",
+                    "cassandra-stress write no-warmup cl=QUORUM n=1000M -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3) compaction(strategy=IncrementalCompactionStrategy)' -mode cql3 native -rate threads=300 -col 'n=FIXED(5) size=FIXED(200)' -pop seq=1000M..2000M",
```

Comment:
1. CL should be ALL for prepare.
2. Do not specify compaction strategy, so we can run it also on OSS, for Enterprise the default is ICS.
3. let's not change for now the default column number and size and stick with the default.

Reply:
updated